org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/color/DocumentColorProvider.java

```diff
@@ -56,30 +61,31 @@ private CompletableFuture<List<? extends ICodeMining>> provideCodeMinings(@NonNu
 			final var textDocumentIdentifier = LSPEclipseUtils.toTextDocumentIdentifier(docURI);
 			final var param = new DocumentColorParams(textDocumentIdentifier);
 			final var colorResults = Collections.synchronizedList(new ArrayList<ColorInformationMining>());
```

Comment:
as a side note, this variable was used for synchronization before, no after this PR, it is unused. I have removed it already in a separate commit, but I would like to make you aware, @ahmedneilhussain , just in case it is important and I have missed it.

Reply:
No, you're quite right - I removed this in the original version but overlooked it in this PR. Apologies! Unfortunately the code has diverged from the original to the extent that I am having to do a "manual cherry-pick"...

Thanks for the correction.